src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -317,19 +329,19 @@ Containers::Optional<MeshData3D> TinyGltfImporter::doMesh3D(const UnsignedInt id
                 return Containers::NullOpt;
             }
 
-            const size_t numPositions = bufferView.byteLength/sizeof(Vector3);
+            const size_t numPositions = accessor.count;
```

Comment:
I think this variable is not really needed anymore. (Yes, sorry, this originates from my patch, which was just hastily put together to make things working).